documentation/src/main/markdown/integrations/jenkinsplugin/downloadingandinstalling.md

```diff
@@ -7,7 +7,7 @@ To install the [detect_product_short] for Jenkins plugin, perform the following
 1. Click **Download now and install after restart**. This is the recommendation for installing the plugin.
 1. After restarting Jenkins, confirm that the plugin is successfully installed by navigating to **Manage Jenkins** > **Manage Plugins > Installed**, and verify that **[company_name] [solution_name]** displays in the list.
 
-[detect_product_short] plugin for Jenkins GitHub page [jenkinsci](https://github.com/jenkinsci/synopsys-detect-plugin).
+[detect_product_short] plugin for Jenkins GitHub page [jenkinsci](https://github.com/jenkinsci/detect-plugin).
```

Comment:
@sig-cpotts If possible, lets handle all integrations related changes in a completely independent PR specific to all integrations section. This repo is in outside organization jenkinsci as you know, and we don't know yet what name will be allowed by the Jenkins infra team.

Reply:
@dmamidisynopsys is it ok if we remove the "synopsys" part for now and then circle back with a new PR after to correct the url as needed?

Reply:
Sure we can. The problem is these won't get updated by end of October most likely, and jenkinsci/ repos are external and we don't have full control on when these might change

Reply:
Ah, ok if https://github.com/jenkinsci/synopsys-detect-plugin will at least still be there until we get the new path, I will leave the previous URL. Thanks.

Reply:
Reverted.